schemas/senv_data.fbs

```diff
@@ -0,0 +1,21 @@
+//NOTE: This schema is under development.
+//If you decide to use it, be aware that backwards compatibility is not guaranteed.
+
+//Used to transmit fast sample environment data
+
+file_identifier "senv";
+
+enum Location : byte { Unknown = 0, Start, Middle, End }
+
+table senv_data {
+    Name: string (required);     // Name of the device plus channel number (e.g. "SomeName_3")
```

Comment:
Channel number is already a field, would it make more sense for this field to be `DeviceName` instead and omit the channel number?

Reply:
Having the channel number as an integer is a (very cheap) nicety that I think we should have. As for the name, there are two options in my opinion:
1. Have the name be specific to the ADC device and let the file writer module construct a unique string from  when overriding `FBSchemaReader::sourcename_impl()`.
2. Have the name be unique to the channel.

I obviously went with the choice 2 as this is what the other file writing modules currently do. I do not have a problem with changing that however.